### PR TITLE
feat: skip processing for positions listed in `zeebe.broker.processing.skipPositions`

### PR DESCRIPTION
**Pull request by lenaschoenburg**
This implements the first half of #16177, allowing us to set the new configuration value `zeebe.broker.processing.skipPositions` in order to skip processing for these commands.

The config value accepts any comma-separated list of `Long`s.

### CODE REVIEW

File: stream-platform/src/main/java/io/camunda/zeebe/stream/impl/SkipPositionsFilter.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ */
+package io.camunda.zeebe.stream.impl;
+
+import io.camunda.zeebe.logstreams.log.LoggedEvent;
+import io.camunda.zeebe.stream.api.EventFilter;
+import java.util.Set;
+import org.agrona.collections.LongHashSet;
+
+/** A filter that skips events with positions in a given set of positions. */
+public final class SkipPositionsFilter implements EventFilter {
```
Comment (deepthidevaki) [inline stream-platform/src/main/java/io/camunda/zeebe/stream/impl/SkipPositionsFilter.java:16]: :+1:

**Review by deepthidevaki — APPROVED** [review]
LGTM. But before merging, could you also add a test for the skipping behavior of StreamProcessor?

File: broker/src/main/java/io/camunda/zeebe/broker/system/configuration/ProcessingCfg.java [hunk 2]
```diff
@@ -37,6 +41,15 @@ public void setEnableAsyncScheduledTasks(final boolean enableAsyncScheduledTasks
     this.enableAsyncScheduledTasks = enableAsyncScheduledTasks;
   }
 
+  public Set<Long> skipPositions() {
+    return skipPositions != null ? skipPositions : Set.of();
+  }
+
+  public void setSkipPositions(final String skipPositions) {
+    this.skipPositions =
+        ConfigurationUtil.parseCsv(skipPositions, Long::parseLong).collect(Collectors.toSet());
```
[the remark this answers is not in the document]
Comment (oleschoenburg) [reply broker/src/main/java/io/camunda/zeebe/broker/system/configuration/ProcessingCfg.java:50]: Well, TIL :facepalm:

**Comment by oleschoenburg** [comment]
Skipping the backport to 8.1 because the StreamProcessor there is fairly different.

**Pull request merged**